src/test/kotlin/calculator/CalculationParameterTest.kt

```diff
@@ -0,0 +1,81 @@
+package calculator
+
+import org.assertj.core.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class CalculationParameterTest {
+
+    @Test
+    fun `calculationParameter 는 연산자, 피연산자를 나누어서 가지고 있다`() {
+        // given
+        val input = "1 + 2"
+
+        // when
+        val calculationParameter = CalculationParameter(input)
+
+        // then
+        assertThat(calculationParameter.operators[0]).isEqualTo(Operator.PLUS)
+        assertThat(calculationParameter.operands).isEqualTo(listOf(1, 2))
+    }
+
+    @Test
+    fun `입력값에 공백이 포함되어 있어도 정상적으로 연산자, 피연산자를 구분할수 있다`() {
+        // given
+        val input = "1   0 + 2    0 -  3   0 /   4  5   * 6  7"
+
+        // when
+        val calculationParameter = CalculationParameter(input)
+
+        // then
+        assertThat(calculationParameter.operators).isEqualTo(
+            listOf(
+                Operator.PLUS, Operator.MINUS, Operator.DIVISION, Operator.MULTIPLICATION
+            )
+        )
+        assertThat(calculationParameter.operands).isEqualTo(listOf(10, 20, 30, 45, 67))
+    }
+
+    @Test
+    fun `CalculationParameter 에 null 을 입력한 경우 에러가 발생한다`() {
+        // given
+        val nullInput = null
+
+        // then
+        assertThatThrownBy {
+            CalculationParameter(nullInput)
+        }.hasMessage(ErrorMessage.NULL_INPUT_ERROR)
```

Comment:
저는 메세지도 검증하지만, 메세지는 자주 변할 수 있기때문에 Exception의 타입도 함께 검증을 하고있습니다! :) 

```suggestion
        assertThatThrownBy {
            CalculationParameter(nullInput)
        }.isInstanceOf(IllegalArgumentException::class.java)
            .hasMessage(ErrorMessage.NULL_INPUT_ERROR)
```

Reply:
알려주셔서 감사합니다! 이렇게 적용하겠습니다~!
`isInstanceOf` 를 넣어서  테스트 코드 작성하는중  에러가 계속 발생하여 적용하지 못했습니다.